### DISCUSSION TITLE
deploy latest tag on master

### DISCUSSION THREAD

**Discussion by arderyp** [deployphp/deployer#3647] [Help needed]
i know there is a `--tag=x.y.z` option. I'm wondering is there is a way to tell deployer something like `dep deploy --tag=latest`, where it would automatically use the most recent tag published on master.  Basically: "deploy the latest release"

I know there are lots of different approaches to querying GH API or leveraging `git` to determine what the latest tag is, and the various approaches seem hotly debated.  So maybe there is no clear answer and thus this must be implemented on a case-by-case basis.

Just thought I'd ask to see what the community thinks.

**Comment by antonmedv** [comment]
Nope. There is no such thing at latest. 

As a one of the solutions: add latest tag with your version tag.

**Reply by arderyp** [reply]
Interesting idea adding latest tag with each release. I guess o then have to delete and death with each release, but that could work.  Thanks!
